docs/cow-protocol/concepts/order-types/pay-debt-flashloans.md

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 6
+---
+
+# Repay debt with collateral using flashloans
+
+A key advantage of flashloans is the ability to repay debt with collateral, since flashloans allow users to close or reduce their debt positions without needing upfront liquidity.
```

Comment:
repaying debt with collateral is not a feature of flashloans but just one use case.